### PR TITLE
Data-loader code in a separate folder.

### PR DESCRIPTION
**Pull request by Akarshit**

### CODE REVIEW

File: data-loader/data/images.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    });
+}
+
+
+export function getImage(storeName, imageIndex) {
+    return binaryCachedImages[storeName][imageIndex];
+}
+
+/**
+ * @method generateImage
```
Comment (zenweasel) [inline data-loader/data/images.js]: Can we port this function into the main, plugin version of devtools? The colored images are way better than the static ones

File: data-loader/package.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "reaction-developer-tools",
```
Comment (zenweasel) [inline data-loader/package.json]: If we are going to make this a separate package like this, maybe we should call this `dataloader` or something and just make it a read separate package?

**Comment by spencern** [comment]
@zenweasel @Akarshit Can we update or close this PR?

**Comment by zenweasel** [comment]
@Akarshit Is this PR still relevant?

**Comment by Akarshit** [comment]
`data-loader` is now in a separate repo and will be used by `devtools` as a package.

**Pull request closed without merging**